source/utilities.tex

```diff
@@ -8316,15 +8299,16 @@
 throw an exception.
 
 \pnum
-\effects Constructs a \tcode{unique_ptr} by transferring ownership from
-\tcode{u} to \tcode{*this}. If \tcode{D} is a reference type, this
+\effects Constructs a \tcode{unique_ptr} from
+\tcode{u}. If \tcode{D} is a reference type, this
 deleter is copy constructed from \tcode{u}'s deleter; otherwise, this
 deleter is move constructed from \tcode{u}'s deleter. \begin{note} The
 deleter constructor can be implemented with \tcode{std::forward<D>}. \end{note}
```

Comment:
(Drive-by:) "The construction of the deleter" would be more correct here.

Reply:
@jensmaurer: ping

Reply:
Fixed.